realm/realm-jni/src/util.hpp

```diff
@@ -72,7 +72,12 @@ JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM *vm, void *reserved);
     catch (util::File::AccessError& e) { \
         ThrowException(env, FileAccessError, string(fileName), \
                 std::string(e.what()) + " path: " + e.get_path()); \
-    }
+    } \
```

Comment:
And `CATCH_FILE` actually should be removed since we have the file info from core exception. I will do it later.